lib/components/molecules/Accordian/Accordian.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+
+import {
+  Accordion as ReactAccordion,
+  AccordionItem,
+  AccordionItemTitle,
+  AccordionItemBody,
+} from 'react-accessible-accordion';
+import styles from './Accordian.style';
+
+type Props = {
+  children: Node,
+  accordionProps: Object,
```

Comment:
can you define the property types for this accordionProps using flow. So we know what all it can receive.